tests/tensorflow/test_models/shared_layers_model.py

```diff
@@ -12,7 +12,7 @@
 """
 
 import tensorflow as tf
-from tensorflow.keras import layers
+from nncf.tensorflow.tf_internals import layers
```

Comment:
Could you please explain why it was changed?
This code
```python
from tensorflow.keras import layers
```
is correct. It is a public API. Please see [this](https://www.tensorflow.org/versions/r2.8/api_docs/python/tf/keras/layers) for more details.

Reply:
In our develop code, some are using `tf.python.keras.layers` and some are using `tf.keras.layers` so it was for consistency. Modified it

Reply:
Just to be clear. Let's take a look at the `Rescaling` layer. We can't find this layer for TF < 2.8 in the `tf.keras.layers`.
So for this reason we use `tensorflow.python.keras.layers` for TF < 2.6 and `keras.layers` for TF > 2.6.
This is used only in tests (for historical reasons). It is not good practice. We always should use public API (i.e. `tf.keras.layers`).